monitoring/controllers/grafana-mcp/release.yaml

```diff
@@ -0,0 +1,95 @@
+---
+apiVersion: helm.toolkit.fluxcd.io/v2
+kind: HelmRelease
+metadata:
+  name: grafana-mcp
+  namespace: monitoring
+spec:
+  interval: 1h
+  chart:
+    spec:
+      chart: grafana-mcp
+      version: ">=0.1.0"
+      sourceRef:
+        kind: HelmRepository
+        name: grafana-charts
+        namespace: flux-system
+      interval: 60m
+  dependsOn:
+    - name: kube-prometheus-stack
+  values:
+    # Official Grafana MCP Server image from Docker Hub
+    image:
+      repository: mcp/grafana  # {"$imagepolicy": "flux-system:grafana-mcp-policy:name"}
+      tag: ""  # {"$imagepolicy": "flux-system:grafana-mcp-policy:tag"}
+      registry: docker.io
+
+    # Grafana connection configuration
+    # Environment variables are set via envFrom from sealed secret
+    grafana:
+      url: ""  # Set via GRAFANA_URL from sealed secret
```

Comment:
The Grafana URL is not sensitive information and is a fixed value because the kube-prometheus-stack deploys it in the same cluster.
Remove it from the secret and fix it in the release